sources/packages/web/src/services/http/dto/BypassedRestriction.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { RestrictionType } from "@/types";
+
+/**
+ * Base DTO for bypassed restriction
+ */
+export interface BypassedRestrictionBaseDTO {
+  id: number;
+  restrictionType: RestrictionType;
+  restrictionCode: string;
+}
+
+/**
+ * DTO interface for student/institution bypassed restriction summary.
+ */
+export interface BypassedRestrictionSummaryAPIOutDTO
```

Comment:
Please follow the comment from API to create a wrapper DTO to wrap the array result.

Reply:
Updated.